### PR TITLE
test: add the test case for org.dromara.soul.plugin.sync.data.webosck…

### PR DESCRIPTION
**Pull request by cocoZwwang**
…et.handler.AuthDataHandler

Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-sync-data-center/soul-sync-data-websocket/src/test/java/org/dromara/soul/plugin/sync/data/weboscket/handler/AuthDataHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Test cases for {@link AuthDataHandler}.
+ *
+ * @author cocoZwwang
+ */
+public class AuthDataHandlerTest {
```
Comment (songyuequan) [inline soul-sync-data-center/soul-sync-data-websocket/src/test/java/org/dromara/soul/plugin/sync/data/weboscket/handler/AuthDataHandlerTest.java:38]: add final on the test class

**Review by songyuequan — CHANGES_REQUESTED** [review]

File: soul-sync-data-center/soul-sync-data-websocket/src/test/java/org/dromara/soul/plugin/sync/data/weboscket/handler/AuthDataHandlerTest.java (hunk 1 above)
Comment (cocoZwwang) [reply soul-sync-data-center/soul-sync-data-websocket/src/test/java/org/dromara/soul/plugin/sync/data/weboscket/handler/AuthDataHandlerTest.java]: OK! I've modified and submitted.

**Review by tydhot — APPROVED** [review]
the code annotation is cool!

**Comment by cocoZwwang** [comment]
> the code annotation is cool!

Thanks!

**Pull request merged**